libraw/src/rawimage.rs

```diff
@@ -0,0 +1,37 @@
+use crate::{Processor, Sizes};
+use std::ops::Deref;
+use std::slice;
+
+use libraw_sys as sys;
+
+pub struct RawImage {
+    _processor: Processor,
+    rawdata: *const sys::libraw_rawdata_t,
+    pub sizes: Sizes,
```

Comment:
I'd prefer this to be a getter, building `Sizes` from `processor.inner.sizes` when needed. It should be a `Copy` type anyway, so it's a very fast operation.